puppeteer/bundleRequests/index.test.js

```diff
@@ -21,28 +18,37 @@ const onPageRequest = request => {
   context.page.__requests.push(request.url());
 };
 
-const testSuites = Object.keys(config)
-  .map(serviceId => {
-    return Object.keys(config[serviceId].pageTypes)
-      .filter(
-        pageType =>
-          shouldSmokeTest(pageType, serviceId) &&
-          serviceHasPageType(serviceId, pageType),
-      )
-      .map(pageType => {
-        const paths = getPaths(serviceId, pageType);
-
-        const [path] = paths;
-
-        return {
-          path,
-          pageType,
-          service: config[serviceId].name,
-          runforEnv: ['local'],
-          tests: [assertLoadsExpectedBundles, assertLoadsModernBundles],
-        };
-      });
-  })
-  .flat();
+const testSuites = [
+  {
+    path: '/pidgin',
+    pageType: 'homepage',
+    service: 'pidgin',
+    runforEnv: ['local'],
+    tests: [assertLoadsExpectedBundles, assertLoadsModernBundles],
+  },
+  {
+    path: '/serbian/cyr',
+    pageType: 'homepage',
+    service: 'serbian',
+    runforEnv: 'local',
+    variant: 'cyr',
+    tests: [assertLoadsExpectedBundles, assertLoadsModernBundles],
+  },
+  {
+    path: '/serbian/lat',
+    pageType: 'homepage',
+    service: 'serbian',
+    runforEnv: 'local',
+    variant: 'lat',
+    tests: [assertLoadsExpectedBundles, assertLoadsModernBundles],
+  },
+  {
+    path: '/news/articles/c0g992jmmkko',
```

Comment:
Should we include news here, given that we don't render canonical articles, only AMP..? I'm inclined to say, let's remove it for now. If we don't have a ticket, could we add one, to revisit these tests & add more test coverage.